### DISCUSSION TITLE
Multiple recognition APIs?

### DISCUSSION THREAD

**Discussion by lonix1** [SeaDve/Mousai#141] [Q&A] [answered]
Are there are alternatives to Audd, which allow more API calls in their free tiers?

Maybe in the future the app can use multiple services.

**Comment by SeaDve** [comment, marked as answer]
Mousai already supports multiple recognition APIs, e.g., Shazam, though behind a dev panel.

**Comment by lonix1** [comment]
Will it be exposed in the stable build someday?

**Reply by SeaDve** [reply]
Yes, it will. Just haven't got the motivation and time yet to do it
